pisa-proxy/protocol/mysql/src/server/packet.rs

```diff
@@ -14,11 +14,11 @@
 
 use std::{
     io::{Error, ErrorKind},
-    ptr::copy_nonoverlapping,
+    ptr::copy_nonoverlapping, u8,
```

Comment:
if `u8` and `net::TcpStream` is not used, remove it.

Reply:
fixed